test/onnx/test_fx_op_consistency.py

```diff
@@ -188,6 +188,9 @@
         "addmm", dtypes=onnx_test_common.BOOL_TYPES,
         reason=onnx_test_common.reason_onnx_does_not_support("Addmm")
     ),
+    xfail(
+        "all", reason=onnx_test_common.reason_onnx_script_does_not_support("aten.all.dims")
```

Comment:
```suggestion
        "all", reason=onnx_test_common.reason_onnx_script_does_not_support("aten.all.dims") + "https://github.com/pytorch/pytorch/issues/XXXXX"
```

Maybe add the tracking issue as part of the reason or a `# TODO: Remove after https://github.com/pytorch/pytorch/issues/XXXXX` comment around it?

@justinchuby how do you feel changing `this to def reason_onnx_script_does_not_support(operator, dtypes, github_issue=None)` to help tracking the underlying issue. That is a way to remind users to create issues without our request

Reply:
I will update https://github.com/pytorch/pytorch/pull/110536